File: TransCoder_tokenized_test_set_functions/python/SURVIVAL.py
def survival ( S, N, M ) :
    if ( ( ( N * 6 ) < ( M * 7 ) and S > 6 ) or M > N ) :
        print ( "No" )
    else :
        days = ( M * S ) / N
        if ( ( ( M * S ) % N ) != 0 ) :
            days += 1
        print ( "Yes " ),
        print ( days )
    

File: TransCoder_tokenized_test_set_functions/java/SURVIVAL.java
static void survival ( int S, int N, int M ) {
  if ( ( ( N * 6 ) < ( M * 7 ) && S > 6 ) || M > N ) System . out . println ( "No" );
  else {
    int days = ( M * S ) / N;
    if ( ( ( M * S ) % N ) != 0 ) days ++;
    System . out . println ( "Yes " + days );
  }
}